### DISCUSSION TITLE
where is is possible to get a uPython board from nowadays?

### DISCUSSION THREAD

**Discussion by grodstein** [micropython/micropython#11102] [Hardware & Peripherals] [answered]
Where is it possible to buy a board running MicroPython nowadays? When I go to https://store.micropython.org, every single board is sold out, with no hints as to availability. Presumably this is still due to a chip shortage (digikey.com confirms no availability of any of the MCUs) -- but is there a workaround while we wait for parts to be available at some undetermined date? Do most people buy an Arduino or other board and then re-flash it as MicroPython?

Background: I'm starting to prepare to teach a course in September in building embedded medical devices, and am deciding whether it's reasonable to do it with MicroPython. But if there are no boards, then the answer is clear :-(
Thanks,
/Joel

**Comment by peterhinch** [comment, marked as answer]
There are plenty of platforms other than STM. For many applications the Pico and Pico W are highly cost effective; the ESP32 variants are also worth a look. See [the download page](http://micropython.org/download/) to get an idea of available ports.

**Reply by grodstein** [reply]
Makes sense, thanks.
/Joel

**Comment by mattytrentini** [comment]
@grodstein there are a couple of us that use MicroPython professionally for medical device development, please reach out if we can help!

**Reply by grodstein** [reply]
thanks! I will for sure.
/Joel
